### ISSUE TITLE
IoTivity node support for Android

### ISSUE THREAD

**Issue by pramodhg123** [intel/iotivity-node#173]
Need support for enabling the IoTivity node js on Android.

**Comment by gabrielschulhof** [comment]
@pramodhg123 do you mean that Node.js will be running on Android, and iotivity-node will be loaded within it?

**Comment by pramodhg123** [comment]
Yes node js will be be running in Android and iotivity-node runs inisde it

Regards
Pramod H G
